test/Distributed/distributed_actor_system_missing_adhoc_requirement_impls.swift

```diff
@@ -762,14 +762,11 @@ struct FakeInvocationEncoder_recordArgument_wrongType: DistributedTargetInvocati
   mutating func doneRecording() throws {}
 }
 struct FakeInvocationEncoder_recordArgument_missingMutating: DistributedTargetInvocationEncoder { 
-  //expected-error@-1{{type 'FakeInvocationEncoder_recordArgument_missingMutating' does not conform to protocol 'DistributedTargetInvocationEncoder'}}
+  //expected-error@-1{{struct 'FakeInvocationEncoder_recordArgument_missingMutating' is missing witness for protocol requirement 'recordArgument'}}
```

Comment:
I was debugging why the hell only the recordArgument changed behavior here and could not figure it out so far... all the checks are the same after all hm. May be a clue, I don't know yet.